### PR TITLE
restructure the workspace docs

### PR DESCRIPTION
**Pull request by cwyl02**
This is the prerequisite for the Operator docs PRs.

### CODE REVIEW

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: layout.pug
+navigationTitle: Workspace Catalog Applications
+title: Workspace Catalog Applications
+menuWeight: 20
+excerpt: Workspace Catalog Applications
+---
+<!-- TODO: settle down the link target to custom applications -->
+Catalog applications are any third-party or open source applications that appear in the Catalog. These applications can be provided by D2iQ for use in your environment, or [Custom Applications](../../../projects/applications/catalog-applications/custom-applications/) that can be used but are not supported by D2iQ.
```
Comment (cbuto) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md:9]: I wonder if we should leave out `custom applications` in the workspace for now

File: pages/dkp/kommander/2.1/workspaces/applications/platform-applications/application-deployment/index.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ export WORKSPACE_NAMESPACE=<workspace_namespace>
 
 ## Deploy the application
 
-The list of available applications that can be deployed on the attached cluster can be found [here](../../workspace-platform-services#workspace-platform-applications).
```
Comment (cwyl02) [inline pages/dkp/kommander/2.1/workspaces/applications/platform-applications/application-deployment/index.md:32]: @mesosphere/d2iq-docs FYI for the `here`link change

File: pages/dkp/kommander/2.1/workspaces/applications/platform-applications/index.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ This takes you to the **Applications** page which displays all applications that
 
 To use the CLI to deploy or uninstall applications, see [Application Deployment](./application-deployment)
 
-<p class="message--important"><strong>IMPORTANT: </strong>There may be dependencies between the applications, which are listed <a href="./platform-service-dependencies/">here</a>. Review them carefully prior to customizing to ensure that the applications are deployed successfully.</p>
```
Comment (cwyl02) [inline pages/dkp/kommander/2.1/workspaces/applications/platform-applications/index.md:28]: @mesosphere/d2iq-docs FYI for the herelink change

**Review by cbuto — APPROVED** [review]

**Review by Colin1841 — APPROVED** [review]

**Comment by cwyl02** [comment]
Vale is failing due to Github Action configuration issue. merging

**Pull request merged**